controllers/contacts.js

```diff
@@ -0,0 +1,110 @@
+const Contact = require("../models/contact");
+
+const { HttpError } = require("../helpers/httpError");
+
+const {
+  addContactValidationSchema,
+  addFavoriteValidatedSchema,
+} = require("../utils/validation/contactValidationSchemas");
+
+async function getAll(req, res, next) {
```

Comment:
```suggestion
async function getAllContacts(req, res, next) {
``` аналогічно для інших контроллерів